### PR TITLE
Fix action button text for long translations

### PR DESCRIPTION
**Pull request by GretaD**
the problem
before_longtezt

### CODE REVIEW

**Comment by GretaD** [comment]
@skjnldsv do you think this approach is good? This happens with different languages that have long translation for a certain text, and `white-space: pre-wrap` doesnt work.

**Comment by skjnldsv** [comment]
I think it could make sense to enlarge a bit, but it might be just an issue with the title not having a width 100% and being ellipsised properly instead 🤔

**Comment by GretaD** [comment]
> I think it could make sense to enlarge a bit, but it might be just an issue with the title not having a width 100% and being ellipsised properly instead thinking

I tried adding `width: 100%` that also needs a `display: block` on __title. But still doesnt work.

**Comment by skjnldsv** [comment]
> I tried adding `width: 100%` that also needs a `display: block` on __title. But still doesnt work.

You need a bit more :)
```css
	text-overflow: ellipsis;
	overflow: hidden;
	white-space: nowrap;
	max-width: 100%;
	display: inline-block;
```

**Comment by skjnldsv** [comment]
Capture d’écran_2020-11-17_19-36-36

File: src/assets/action.scss [hunk 1]
```diff
@@ -90,7 +90,7 @@
 
 		// long text area
 		p {
-			width: 150px;
+			width: 185px;
```
Comment (skjnldsv) [inline src/assets/action.scss:93]: Make sure @jancborchardt is ok with this :)

**Review by skjnldsv — APPROVED** [review]

**Comment by GretaD** [comment]
I also increased a bit the `p` width, to make sure to ellips only very long translations and not everything that is a bit longer than english

**Review by juliushaertl — APPROVED** [review]

File: src/assets/action.scss (hunk 1 above)
Comment (juliushaertl) [reply src/assets/action.scss:93]: :ping_pong: @jancborchardt

**Pull request merged**